zkevm-circuits/src/super_circuit.rs

```diff
@@ -318,6 +358,22 @@ impl<F: Field> SubCircuit<F> for SuperCircuit<F> {
     /// Returns suitable inputs for the SuperCircuit.
     fn instance(&self) -> Vec<Vec<F>> {
         let mut instance = Vec::new();
+
+        let block = self.block.as_ref().unwrap();
+
+        instance.extend_from_slice(&[
```

Comment:
Because to check consecutive chunks context via instance commitment, and treat instance commitment as blackbox, 
- [current_chunk_index, total_chunk, initial_rw] 
- [next_chunk_index, total_chunk, end_rw]
We need to put `total_chunk` in both context column so the equality check is satisfied.